src/providers/info.rs

```diff
@@ -68,6 +75,16 @@ impl InfoProvider for DefaultInfoProvider<'_> {
     }
 }
 
+impl DefaultInfoProvider<'_> {
+    pub fn exchange_usd_to_eur(&self) -> Result<f64> {
+        let url = String::from("https://api.exchangeratesapi.io/latest?base=USD&symbols=EUR");
```

Comment:
As mentioned above I would not limit this to EUR ... I am not sure where this requirement that we only support `EUR` is comming from. We want to support all currencies that we have an exchange rate for (see how it is currently in the web interface)

Reply:
From nowhere. I think I just misunderstood the ticket and assumed we just wanted that.